src/server/status/metrics.js

```diff
@@ -1,39 +1,71 @@
-import _ from 'lodash';
+import { get, set, isObject } from 'lodash';
 import { keysToSnakeCaseShallow } from '../../utils/case_conversion';
+import { getAllStats as cGroupStats } from './cgroup';
+
+let cGroupStatsAvailable = true;
 
 export function collectMetrics(kbnServer, server, config) {
-  server.plugins['even-better'].monitor.on('ops', function (event) {
-    kbnServer.metrics = getMetrics({ event, config });
+  server.plugins['even-better'].monitor.on('ops', event => {
+    getMetrics(event, config, server).then(data => { kbnServer.metrics = data; });
   });
 }
 
-export function getMetrics({ event, config }) {
+export async function getMetrics(event, config, server) {
   const port = config.get('server.port');
   const timestamp = new Date().toISOString();
```

Comment:
do you think we should set timestamp after cgroup stats have been read?  along the same lines, do you think we should only getMetrics once the previous getMetrics call is done so they don't queue up for frequent ops events?